isc24/CernVM-FS/outline.tex

```diff
@@ -56,18 +56,16 @@ \subsection*{Outline}
  
 %===============================================================================
 
-\subsubsection*{Half-day format, 3.5 hours of tutorial content}
-
 \begin{itemize}[style=standard, labelwidth=1.9cm]
 
     \item [00:00-00:30] \emph{(30min)} Introduction to CernVM-FS
     \item [00:30-00:50] \emph{(20min)} Example CernVM-FS repository: EESSI (incl. hands-on usage)
-    \item [00:50-02:00] \emph{(70min)} Accessing CernVM-FS repositories (hands-on)
-    \begin{itemize}
-        \item Installing and configuring CernVM-FS client system (hands-on)
-        \item Installing and configuring proxy server (demo) + re-configuring client (hands-on)
-        \item Installing and configuring mirror server (demo) + re-configuring client (hands-on)
-    \end{itemize}
+    \item [00:50-02:00] \emph{(70min)} Accessing CernVM-FS repositories (hands-on), including:
+%    \begin{itemize}
+%        \item[]- \emph{(30min)} Installing + configuring CernVM-FS client system (hands-on)
+%        \item[]- \emph{(20min)} Installing + configuring proxy server (demo), re-configuring client (hands-on)
+%        \item[]- \emph{(20min)} Installing + configuring mirror server (demo), re-configuring client (hands-on)
+%    \end{itemize}
```

Comment:
The list is omitted?

Reply:
Yes, otherwise we exceed the page limit. The breakdown is also provided in the description section, which should be good enough.